src/components/MultipleAvatars.js

```diff
@@ -63,6 +68,15 @@ const MultipleAvatars = (props) => {
         );
     }
 
+    if (props.showSubscript) {
+        return (
+            <SubscriptAvatar
+                avatarImageURLs={props.avatarImageURLs}
+                avatarTooltips={props.avatarTooltips}
+            />
+        );
+    }
```

Comment:
I had a question about why we should use `MultipleAvatars`  component just to return a different component when a boolean prop is present? Does it make sense to just use this component directly?

Reply:
Oh not having dependency with `MultipleAvatars` makes sense, I removed it from `MultipleAvatars` and call it directly.